### PR TITLE
DevTools 112: Recorder extensions: replay ext support, lighthouse opt, get opt

### PR DESCRIPTION
**Pull request by sofiayem**
Documented:
- https://developer.chrome.com/blog/new-in-devtools-112/#replay-extensions
- https://developer.chrome.com/blog/new-in-devtools-112/#puppeteer-lighthouse
- https://developer.chrome.com/blog/new-in-devtools-112/#get-extensions

Publish on Apr 4

### CODE REVIEW

File: site/en/docs/devtools/recorder/reference/index.md [hunk 1]
```diff
@@ -63,33 +64,50 @@ You can export and import user flows in the Recorder. This is useful for bug rep
...
 
 {% Aside %}
 **Note**: This feature is available from Chrome version 104.
 {% endAside %}
 
-You can install a Chrome extension to export replay scripts in your favorite format. 
+To integrate the **Recorder** with your tools, install extensions:
+
+1. Select {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/4dU9UXvsinS4zbgjd8rK.svg", alt="File download.", width="20", height="20" %} **Export** > **Get extensions...** at the top of the **Recorder** panel or go to [Extensions](/docs/devtools/recorder/extensions/).
+1. Choose an extension an click its link.
```
Comment (rachelandrew) [inline site/en/docs/devtools/recorder/reference/index.md:102]:
```suggestion
1. Choose an extension and click its link.
```

File: site/en/docs/devtools/recorder/reference/index.md [hunk 2]
```diff
@@ -100,9 +118,12 @@ If you don't see the export option after installing the extension, do the follow
 {% Aside 'gotchas' %} 
 **Advanced use case: Build an extension**
 
-You can build your own Recorder extension too. See the [Recorder extension API](/docs/extensions/reference/devtools_recorder/) documentation to learn how to build one.
+To further customize the **Recorder** to your needs, you can build your own extension too:
```
Comment (rachelandrew) [inline site/en/docs/devtools/recorder/reference/index.md:121]:
```suggestion
To further customize the **Recorder** to your needs, you can build your own extension:
```

**Review by rachelandrew — APPROVED** [review]
Just a couple of tiny things.

**Review by rachelandrew — APPROVED** [review]

File: site/en/docs/devtools/recorder/reference/index.md [hunk 3]
```diff
@@ -63,33 +64,50 @@ You can export and import user flows in the Recorder. This is useful for bug rep
...
+- **JSON**. Edit the human-readable JSON object and [import](#import-flows) the JSON file back to the **Recorder**.
+- **@puppeteer/replay**. Replay the script with the [Puppeteer Replay](https://github.com/puppeteer/replay) library. When exporting as a @puppeteer/replay script, the steps remain a JSON object. This option is perfect if you want to integrate with your CI/CD pipeline but still have the flexibility to edit the steps as JSON, later convert and import them back into the **Recorder**.
+- **Puppeteer script**. Replay the script with [Puppeteer](/docs/puppeteer/). Because the steps are converted into JavaScript, you can have more fine-grained customization, for example, looping the steps. One caveat, you can't import this script back into the **Recorder**.
+- **Puppeteer (including Lighthouse analysis)**. This export option is the same as the previous one but it includes a [Lighthouse](/docs/lighthouse/) analysis.
```
Comment (jecfish) [inline site/en/docs/devtools/recorder/reference/index.md:82]:
```suggestion
- **Puppeteer (including Lighthouse analysis)**. This export option is the same as the previous one but it includes a [Lighthouse](/docs/lighthouse/) analysis.
```
Is "includes a Lighthouse analysis" the correct term here? It doesn't include an analysis report, but code to excute and generate one.

**Review by jecfish — APPROVED** [review]
LGTM % comments

**Pull request merged**